.github/no-response.yml

```diff
@@ -0,0 +1,13 @@
+# Configuration for probot-no-response - https://github.com/probot/no-response
+
+# Number of days of inactivity before an Issue is closed for lack of response
+daysUntilClose: 14
+# Label requiring a response
+responseRequiredLabel: more-information-needed
```

Comment:
We should make sure this label (exact text) is already created in this repository.

Reply:
Got it. Will add